.editorconfig

```diff
@@ -16,4 +16,7 @@ indent_size=3
 indent_size=3
 
 [CompositeSequence.hbm.xml]
-indent_size=3
\ No newline at end of file
+indent_size=3
+
+[applicationContext-hibernate.xml]
+indent_style=tab
```

Comment:
Ideally, we would fix the indentation of the files mentioned in this EditorConfig file and rely solely on IntelliJ's configuration.